eessi-2023.06-eb-4.8.2-2023a.yml

```diff
@@ -0,0 +1,2 @@
+easyconfigs:
+  - CUDA-Samples-12.1-GCC-12.3.0-CUDA-12.1.1.eb
```

Comment:
We need to use the easyconfig from https://github.com/easybuilders/easybuild-easyconfigs/pull/19189 here

Reply:
```suggestion
  - CUDA-Samples-12.1-GCC-12.3.0-CUDA-12.1.1.eb:
       # use easyconfig that only install subset of CUDA samples,
       # to circumvent problem with nvcc linking to glibc of host OS;
       # see https://github.com/easybuilders/easybuild-easyconfigs/pull/19189
       options:
         from-pr: 19189